server/webapp/WEB-INF/rails/webpack/models/shared/plugin_infos_new/extensions.ts

```diff
@@ -14,11 +14,14 @@
  * limitations under the License.
  */
 
+import {Stream} from "mithril/stream";
 import {AnalyticsCapabilities} from "./analytics_plugin_capabilities";
 import {AuthCapabilities} from "./authorization_plugin_capabilities";
 import {ElasticPluginCapabilities} from "./elastic_plugin_capabilities";
 import {ExtensionType} from "./extension_type";
 
+const Stream = require("mithril/stream");
```

Comment:
isn't this variable definition conflicting?
As per [this](https://github.com/MithrilJS/mithril.d.ts#stream-example) example, can we use `stream` for the real stream? and `Stream` for the type?

Reply:
The import as per the example wasn't working, it give a `no default export` error. But I've changed the require statement's stream to a small 's'